packages/golang1.13.5/spec

```diff
@@ -0,0 +1,7 @@
+---
+name: golang1.13.5
```

Comment:
Can we just rename the package to `golang1` or something? (For the same reason as above.)